src/Tokenizers/PHP.php

```diff
@@ -1080,6 +1132,7 @@ protected function tokenize($string)
                 && PHP_VERSION_ID >= 50500
                 && $tokenIsArray === true
                 && $token[0] === T_YIELD
+                && isset($this->tstringContexts[$finalTokens[$lastNotEmptyToken]['code']]) === false
```

Comment:
I'm not sure why this was added here. Removing it doesn't show any failing tests and I can't think of a code sample that would require this here.

Reply:
I'm not sure either. Removed.